### PR TITLE
linking release snoozing to settings when release tracking is not set up

### PR DESCRIPTION
**Pull request by ehfeng**
Make it obvious on the next step on how to get release tracking set up, so they can snooze until next release.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/groupDetails/actions.jsx [hunk 1]
```diff
@@ -134,7 +135,7 @@ const GroupActions = React.createClass({
                       <div className="help-text">{t('Snooze notifications until this issue reoccurs in a future release.')}</div>
                     </a>
                   :
-                    <a className="disabled tip" title={t('You need to send release data to Sentry in order to use this feature.')}>
+                    <a href={releaseTrackingUrl} className="disabled tip" title="Set up release tracking in order to use this feature.">
```
Comment (dcramer) [inline src/sentry/static/sentry/app/views/groupDetails/actions.jsx]: use title={t('...')} to ensure its localized still

Comment (ehfeng) [inline src/sentry/static/sentry/app/views/groupDetails/actions.jsx]: Done.

File: src/sentry/static/sentry/app/views/groupDetails/actions.jsx [hunk 2]
```diff
@@ -96,6 +96,7 @@ const GroupActions = React.createClass({
     }
 
     let hasRelease = group.tags.filter(item => item.key === 'release').length;
+    let releaseTrackingUrl = '/' + this.getOrganization().slug + '/' + this.getProject().slug + '/settings/release-tracking/'
```
Comment (mattrobenolt) [inline src/sentry/static/sentry/app/views/groupDetails/actions.jsx]: Missing semi-colon.

This is causing tests to fail.

File: src/sentry/static/sentry/app/views/groupDetails/actions.jsx (hunk 1 above)
Comment (mattrobenolt) [inline src/sentry/static/sentry/app/views/groupDetails/actions.jsx]: You forget to push? :)

Comment (ehfeng) [inline src/sentry/static/sentry/app/views/groupDetails/actions.jsx]: Naw, just had to run.

File: src/sentry/static/sentry/app/views/groupDetails/actions.jsx (hunk 2 above)
Comment (ehfeng) [inline src/sentry/static/sentry/app/views/groupDetails/actions.jsx]: Thanks!

**Pull request merged**